### PR TITLE
Add Authentication

### PR DESCRIPTION
**Pull request by indocomsoft**

### CODE REVIEW

**Comment by indocomsoft** [comment]
TODO: document in swagger

File: .rubocop.yml [hunk 1]
```diff
@@ -26,6 +26,10 @@ Metrics/ClassLength:
   Exclude:
     - 'spec/**/*'
 
+Metrics/LineLength:
```
Comment (sciffany) [inline .rubocop.yml:29]: This is cheating!!!

File: .travis.yml [hunk 2]
```diff
@@ -20,6 +20,7 @@ before_script:
   - psql -c 'create database pypagao_test;' -U postgres
   - cp config/database.yml.travis config/database.yml
   - gem install bundler
+  - bundle exec rails db:setup RAILS_ENV=test
```
Comment (sciffany) [inline .travis.yml:23]: Why is this necessary?

**Review by sciffany — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (indocomsoft) [reply .travis.yml:23]: Cos why not?

**Review by sciffany — DISMISSED** [review]

**Comment by indocomsoft** [comment]
Gonna implement inhouse JWT authentication

**Comment by sciffany** [comment]
Yes!

**Pull request closed without merging**